README.md

```diff
@@ -4,8 +4,84 @@
 
 **Standard Exit Format for L2s built on EVM chains**
 
-An exit format allows calldata to specify how assets locked up and redistributed in an L2 should be paid out in L1. We present a general standard for such a format, along with coders written in Typescript and Cairo, which will aid L2 interoperability and support arbitrary tokens
+We present a general standard for such a format, along with coders written in Typescript and Cairo, which will aid L2 interoperability and support arbitrary tokens
```

Comment:
```suggestion
We present a general standard for such a format, along with coders written in Typescript, which will aid L2 interoperability and support arbitrary tokens.
```

No encoding / decoding of bytes data in Cairo. The description of the Cairo work the the below paragraph is good - I'll check `/cairo/readme.md` for reasonableness.